main.js

```diff
@@ -151,13 +115,171 @@ const inventory = [
     screenQuality: 'Full HD',
     smartTv: false,
     options: {
-      wifi: false,
-      speech: false,
-      hdr: true,
-      bluetooth: false,
-      ambiLight: false,
+        wifi: false, speech: false, hdr: true, bluetooth: false, ambiLight: false,
     },
     originalStock: 10,
     sold: 8,
-  },
-];
\ No newline at end of file
+},];
+
+//opdracht-1//
+
+console.log("-----1A-----")
+
+const tvNames = inventory.map((inventoryType) => {
+    return ({"type": inventoryType.type, "name": inventoryType.name})
+})
+
+console.log(tvNames)
+
+console.log("-----1B-----")
+
+const tvSoldOut = inventory.filter((inventoryEmpty) => {
+    return inventoryEmpty.originalStock === inventoryEmpty.sold
+})
+
+console.log(tvSoldOut)
+
+console.log("-----1C-----")
+
+const tvAmbiLightList = inventory.filter((inventoryAL) => {
+    return inventoryAL.options.ambiLight === true
```

Comment:
Hier levert `inventoryAL.options.ambiLight` ook al een boolean op. Daarom kun je hier `=== true` weglaten
```suggestion
    return inventoryAL.options.ambiLight;
```